Gemfile

```diff
@@ -4,3 +4,9 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in pagerduty.gemspec
 gemspec
+
+group :test do
+  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.6")
+    gem "rspec-given", "< 3.8.1"
+  end
+end
```

Comment:
It seems later versions of `rspec-given` are not compatible with Ruby prior to version 2.6.

The `Gemfile` is not included in the released gem, or influence its content, so configuring this here will only help in development and CI environments.